package.json

```diff
@@ -32,27 +32,27 @@
     "test:coverage:unit": "npm run test:unit -- --coverage"
```

Comment:
No `version` change?

Reply:
We will manage version and changelog before publishing.